test/test_masks_to_bounding_boxes.py

```diff
@@ -0,0 +1,43 @@
+import os.path
+
+import PIL.Image
+import numpy
+import pytest
+import torch
+
+import torchvision.ops
+
+ASSETS_DIRECTORY = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets")
+
+
+@pytest.fixture
+def labeled_image() -> torch.Tensor:
+    with PIL.Image.open(os.path.join(ASSETS_DIRECTORY, "labeled_image.png")) as image:
+        return torch.tensor(numpy.array(image, numpy.int))
+
+
+@pytest.fixture
+def masks() -> torch.Tensor:
+    with PIL.Image.open(os.path.join(ASSETS_DIRECTORY, "masks.tiff")) as image:
+        frames = numpy.zeros((image.n_frames, image.height, image.width), numpy.int)
+
+        for index in range(image.n_frames):
+            image.seek(index)
+
+            frames[index] = numpy.array(image)
+
+        return torch.tensor(frames)
+
+
+def test_masks_to_bounding_boxes(masks):
+    expected = torch.tensor(
+       [[ 127.,    2.,  165.,   40. ],  # noqa: E121, E201, E202, E241
```

Comment:
we should avoid the redundant `.` at the end, i.e. `127.` should just be `127`. We can set the `dtype` argument of the tensor if needed.

Also, let's avoid all these flake8 issues and just format the array as expected.